firstvoices/backend/admin/media_admin.py

```diff
@@ -35,6 +44,26 @@ class ImageAdmin(BaseSiteContentAdmin):
     search_fields = ("title",)
 
 
+@admin.register(EmbeddedVideo)
+class EmbeddedVideoAdmin(BaseSiteContentAdmin, AdminVideoMixin):
+    fields = (
+        "site",
+        "title",
+        "content",
+    )
+    list_display = (
+        "title",
+        "content_url",
+    ) + BaseSiteContentAdmin.list_display
+    search_fields = (
+        "title",
+        "content",
+    )
+
+    def content_url(self, obj):
+        return format_html("<a href='{url}'>{url}</a>", url=obj.content)
```

Comment:
Turns the admin list view content field into clickable links instead of just strings.